README.md

```diff
@@ -14,9 +14,10 @@ A Typescript library for sending Yext Analytics events.
 
 [Full Documentation](./docs/analytics.md)
```

Comment:
Sort of unrelated to the generated docs, but we should probably update this too: 
![Screenshot 2023-08-31 at 12 38 53 PM](https://github.com/yext/analytics/assets/18221975/56e0553e-0aa3-438d-b3b6-d85913c3de23)

Reply:
I don't think either of us have permission to edit it. There would be a gear icon like so if we did.

<img width="86" alt="Screenshot 2023-08-31 at 2 01 15 PM" src="https://github.com/yext/analytics/assets/39343323/943a2671-356d-4968-b16c-d5935a2c24a9">